processing-for-variant-discovery-gatk3.hg38.wgs.inputs.json

```diff
@@ -32,7 +32,10 @@
     "gs://genomics-public-data/resources/broad/hg38/v0/Homo_sapiens_assembly38.known_indels.vcf.gz.tbi"
   ],
 
-  "##_COMMENT4": "DISK SIZES + PREEMPTIBLES",
+  "##_COMMENT4": "Docker",
+  "##GenericPreProcessingWorkflow.gitc_docker": "String? (optional)",
```

Comment:
The workflow has its gitc_docker set to a defult [docker image](https://github.com/gatk-workflows/gatk3-data-processing/blob/a753760a585a962228114be06c465d1f00c64792/processing-for-variant-discovery-gatk3.wdl#L446). 
The docker variable is optional but i left it in the json incase some wants to use an updated version. The line is commented it out because cromwell will through a fit if its not set.

Reply:
got it. Could you then add a comment to the effect of "uncomment the following line and supply a valid docker container to override the default", and also change the "String? (optional)" to something else...it's confusing as it stands (IMO)